### PR TITLE
Adds an example repo for bootstrapping and using shared modules

### PR DESCRIPTION
**Pull request by eefahy**
This is blocked by https://github.com/protocol/infra-team/pull/7 which should get pulled in first

### CODE REVIEW

**Comment by gmasgras** [comment]
screen shot 2018-10-31 at 3 07 54 pm

LGTM ? :)

File: bootstrap_terraform/shared_modules/eks/main.tf [hunk 1]
```diff
@@ -0,0 +1,96 @@
+variable "cluster_name" {}
```
Comment (gmasgras) [inline bootstrap_terraform/shared_modules/eks/main.tf]: won't this be pulled on `terraform init` from `git::https://github.com/protocol/infra-team.git//bootstrap_terraform/shared_modules/eks?ref=8b941b8` ?

File: bootstrap_terraform/shared_modules/vpc/main.tf [hunk 2]
```diff
@@ -1,15 +1,17 @@
 variable "environment" {}
```
Comment (gmasgras) [inline bootstrap_terraform/shared_modules/vpc/main.tf]: also pulled on `terraform init` ?

**Comment by eefahy** [comment]
Whoops I had a plan file checked in +369 lines is much better

**Pull request merged**